SGoopas/Assets/Scripts/MasterMonoBehaviour.cs

```diff
@@ -27,4 +52,24 @@ void Awake()
 		animator.enabled = false;
 		messageContainer.SetActive (false);
 	}
+
+	public void FadeScreen(float target, Action action = null){
+		isFading = true;
+		fadeTarget = target;
+		faderCallback = action;
+	}
+
+	public void SetFade(float a){
```

Comment:
nit: make this guy private

Reply:
This needs to be public. FadeScreen is used for animating while SetFade is used for cutting immediately to a fade value. SetFade is important when fading in scenes. Ex: When the menu opens, I use SetFade to make the screen black, then call FadeScreen to animate it away.